gradle/kotlin-static-analysis.gradle

```diff
@@ -7,7 +7,7 @@ ktlint {
     verbose = true
     android = false
     outputToConsole = true
-    ignoreFailures = false
+    ignoreFailures = true
```

Comment:
what is this for?

Reply:
Lint errors has not been fixed. @saini2sandeep had asked me to make this change get it working